### PR TITLE
RPRBLND-2270: Improve OCIO support, Blender 3.5 upgraded OCIO to version 2.2.0 at the moment RadeonProRenderSDK provide OCIO 2.1.1

### PR DESCRIPTION
**Pull request by VascoPi**
### PURPOSE
Switch to Blender OCIO configs.

### EFFECT OF CHANGE
Removed OCIO config files, use Blender configs instead.

### TECHNICAL STEPS
Removed ocio folder and adjusted create_zip_addon.py.
Remove redundant code.

### CODE REVIEW

**Comment by bsavery** [comment]
What problem is this fixing?  Does the OCIO files in Blender no longer work with RPR in 3.5?  I'm confused.

**Comment by VascoPi** [comment]
> What problem is this fixing? Does the OCIO files in Blender no longer work with RPR in 3.5? I'm confused.

This pull request updates OCIO configs from 1.0 to 2.1.1. just to keep up to date with RPRSDK.
At the moment errors are not observed even if RPR SDK 2.1.1 uses Blender's OCIO configs 2.2.0.

**Comment by bsavery** [comment]
So why do we need our own configs if we can just use Blender's?

**Comment by VascoPi** [comment]
Concerns were only about backward compatibility versions 2.1.1 and 2.2.0, with no explicit warnings/errors.
Should I remove them and try the tests?

**Review by ARozghon — APPROVED** [review]
Approved by QA

File: src/rprblender/properties/render.py [hunk 1]
```diff
@@ -615,19 +615,12 @@ def init_rpr_context(self, rpr_context, is_final_engine=True, use_gl_interop=Fal
...
-                ocio_config_path = os.path.join(bpy.utils.resource_path('LOCAL'),
-                                                'datafiles', 'colormanagement',
-                                                'config.ocio')
-
-            rpr_context.set_parameter(pyrpr.CONTEXT_OCIO_CONFIG_PATH, ocio_config_path)
+            # set ocio config file to blender included one
+            rpr_context.set_parameter(pyrpr.CONTEXT_OCIO_CONFIG_PATH,
+                                      os.path.join(bpy.utils.resource_path('LOCAL'),
+                                                   'datafiles', 'colormanagement',
+                                                   'config.ocio'))
```
Comment (bsavery) [inline src/rprblender/properties/render.py:622]: Does blender include ocio for all versions?

Comment (VascoPi) [reply src/rprblender/properties/render.py:622]: blender 3.5: OCIO 2.2.0, ocio_profile_version: 2
blender 3.4: OCIO 2.1.1, ocio_profile_version: 2
blender 3.1-3.3: OCIO 2.0.0, ocio_profile_version: 2
blender 2.93-3.0: OCIO 2.0.0, ocio_profile_version: 1

**Review by bsavery — APPROVED** [review]

**Review by DagerD — APPROVED** [review]

**Review by bnagirniak — APPROVED** [review]
Looks good

**Pull request merged**